src/umpire/strategy/AllocationStrategy.cpp

```diff
@@ -28,6 +28,13 @@ void* AllocationStrategy::allocate_internal(std::size_t bytes)
   return allocate(bytes);
 }
 
+void* AllocationStrategy::allocate_named(const std::string& UMPIRE_UNUSED_ARG(name), std::size_t UMPIRE_UNUSED_ARG(bytes))
+{
+  UMPIRE_LOG(Error, "AllocationStrategy::allocate_named() not implemented");
```

Comment:
I think this error message should hint at the fact allocate_named() is only supported for shared resources.

Reply:
Okay.  I think I have fixed this now.  If you agree, let's mark this as resolved.

Reply:
Yes, but the message just tells me that `allocate_named` is not implemented. As a user, it would be more helpful if it explained that allocate_named only exists for shared resources.

If you don't think that's appropriate, we need to catch this incorrect usage higher up the chain (i.e. in the Allocator)

Reply:
Done